dali/test/python/test_dali_tf_dataset.py

```diff
@@ -28,28 +28,39 @@
 file_root = os.path.join(test_data_root, 'db', 'coco', 'images')
 annotations_file = os.path.join(test_data_root, 'db', 'coco', 'instances.json')
 
+
+def tensorflow_minor_version():
+    return tf.__version__.split('.')[1]
+
+def compatible_tensorflow():
+    return tensorflow_minor_version() in {'13', '14'}
```

Comment:
Can you add this constrain to the docs too?

Reply:
It's [there](https://github.com/NVIDIA/DALI/blob/master/dali/python/nvidia/dali/plugin/tf.py#L214) already.